src/annotator/mspacy.py

```diff
@@ -64,10 +65,11 @@ def __init__(self, config):
                 sp.require_cpu()
 
         self.config = config["config"]
+        self.config = be.update_dict(self.config)
 
 
 # build the pipeline from config-dict
-class spaCy_pipe(spaCy):
+class spacy_pipe(spacy):
```

Comment:
The activation of processors (or emission of them) for the spacy pipeline is something that needs some cleaning up. I like the way you put it in the spacy notebook at the top. But ultimately this boils down to a philosophical question: 

1. Do we define a language and use a default model for that language as input, with all its default components? 
2. Or do we define a model in the input? And then use the components? What about if the user sets different components than the model?

For user-friendliness, I would go with 
(a) user selects language, ie english;
(b) user selects what type of annotation they want.
In the code, we then default each language to a default model and check if the processors that are set upon user input are available for that model. This will generally be the case as the default model should have all processors.

Let's just build the input for this case. We can add the case "user selects language AND specific model" or "user uses own model" later.